libmcount/mcount.c

```diff
@@ -1802,9 +1797,25 @@ static void mcount_script_init(enum uftrace_pattern_type patt_type)
 	strv_free(&info.cmds);
 }
 
-static __used void mcount_startup(void)
+int mcount_get_pfd(void)
```

Comment:
I think it's better to move this to misc.c file.

Reply:
Thanks. I will updated it soon